src/staticanalysis/bot/static_analysis_bot/cli.py

```diff
@@ -55,18 +36,14 @@ def main(id,
                           required=(
```

Comment:
We should also cleanup the secrets, maybe file an issue so we remember?

Reply:
https://github.com/mozilla/release-services/issues/2085